### PR TITLE
GitHub env

### PR DESCRIPTION
**Pull request by regislon**
Hello, here is my contribution fro the github env. set up. To modify still, but it's a discusion basis. Taken and adapted frm the ENACIT4R template.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/performance.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+  - [ ] **Runnable**
+  - [ ] **Minimal**: reduce runtime wherever possible and remove complicated details that are irrelevant to the issue at hand.
+
+## Description
+
+Please describe the performance issue.
+
+## Benchmarks
+
+How poorly does disdrdb perform?
```
Comment (ghiggi) [inline .github/ISSUE_TEMPLATE/performance.md:23]: Typo: disdrodb

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,108 @@
+# Disdrodb contributing Guide
```
Comment (ghiggi) [inline CONTRIBUTING.md:1]: DISDRODB all uppercase

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,108 @@
...
+
+Hi! Thanks for taking the time to contribute to disdrodb.
+
+You can contribute in many ways :
+
+ - Join the [discussion](https://github.com/ltelab/disdrodb/discussions)
+
```
Comment (ghiggi) [inline CONTRIBUTING.md:8]: Do we want to create a slack channel?

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -0,0 +1,108 @@
...
+
+Before submitting your contribution, please make sure to take a moment and read through the following guidelines
```
Comment (ghiggi) [inline CONTRIBUTING.md:10]: Missing :

File: CONTRIBUTING.md [hunk 5]
```diff
@@ -0,0 +1,108 @@
...
+## Pull Request Guidelines
+
+### We Develop with Github
+We use github to host code, to track issues and feature requests, as well as accept pull requests.
+
+### We Use [Github Flow](https://docs.github.com/en/get-started/quickstart/github-flow), So All Code Changes Happen Through Pull Requests
+Pull requests are the best way to propose changes to the codebase (we use [Github Flow](https://docs.github.com/en/get-started/quickstart/github-flow)). We actively welcome your pull requests:
+
+## Python
+- We follow the pep8 and the python-guide writing style
```
Comment (ghiggi) [inline CONTRIBUTING.md:34]: We might want to set up flake8?

File: CONTRIBUTING.md [hunk 6]
```diff
@@ -0,0 +1,108 @@
...
+
+- If adding a new feature:
+  - Add accompanying test case.
+  - Provide a convincing reason to add this feature. Ideally, you should open a suggestion issue first and have it approved before working on it.
+  - Present your issue in the 'discussion' part of this repo
+
+- If fixing bug:
+  - If you are resolving a special issue, add `(fix #xxxx[,#xxxx])` (#xxxx is the issue id) in your PR title for a better release log, e.g. `update entities encoding/decoding (fix #3899)`.
+  - Provide a detailed description of the bug in the PR. Live demo preferred.
+  - Add appropriate test coverage if applicable.
```
Comment (ghiggi) [inline CONTRIBUTING.md:72]: I would add something to conventions on how to name branches. Here a proposition: 
- Add a reader:  `reader-<institute>-<campaign>`
- Fix a bug: `bugfix-<some_key>-<word>`
- Improve the doc: `doc-<some_key>-<word>`
- Add a new feature: `feature-<some_key>-<word>`
- Refactor some code: `refactor-<some_key>-<word>`
- Optimize some code: `optimize-<some_key>-<word>`

File: CONTRIBUTING.md [hunk 7]
```diff
@@ -0,0 +1,108 @@
...
+We also welcome financial contributions. Please contact us directly.
+
+## Commit Lint
+
+We follow a commit message convention, to have consistent git messages. The goal is to increase readability and ease of contribution
+    - we use [commit-lint](https://github.com/conventional-changelog/commitlint)
+
+## Credits
+
+Thank you to all the people who have already contributed to {YOUR-REPO-NAME} repository!
```
Comment (ghiggi) [inline CONTRIBUTING.md:108]: Missing DISDRODB here

**Review by ghiggi — COMMENTED** [review]
Nice. Just a few stuffs to fix. 
Maybe have a look also at satpy and pysteps projects ;)

File: CONTRIBUTING.md (hunk 3 above)
Comment (charlottegiseleweil) [reply CONTRIBUTING.md:8]: I would advocate to keep the discussions as close as possible to the code / in the same enviro. Gh discussions is really made for that

**Comment by charlottegiseleweil** [comment]
to be replaced by answer to #18 by @regislon

**Pull request closed without merging**